cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -88,51 +107,130 @@ func CreateGceManager(configReader io.Reader) (*GceManager, error) {
 		return nil, err
 	}
 	manager := &GceManager{
-		migs:      make([]*migInformation, 0),
-		service:   gceService,
-		migCache:  make(map[GceRef]*Mig),
-		zone:      zone,
-		projectId: projectId,
+		migs:        make([]*migInformation, 0),
+		gceService:  gceService,
+		migCache:    make(map[GceRef]*Mig),
+		zone:        zone,
+		projectId:   projectId,
+		clusterName: clusterName,
+		mode:        mode,
 		templates: &templateBuilder{
 			projectId: projectId,
 			zone:      zone,
 			service:   gceService,
 		},
 	}
+
+	if mode == ModeGKE {
+		gkeService, err := gke.New(client)
+		if err != nil {
+			return nil, err
+		}
+		manager.gkeService = gkeService
+		err = manager.fetchAllNodePools()
+		if err != nil {
+			glog.Errorf("Failed to fech node pools: %v", err)
```

Comment:
s/fech/fetch

Reply:
Why do we ignore this error? Shouldn't we just consider that a failure to build manager and return the error?

Reply:
Done.